WalletWasabi/Userfacing/PasswordHelper.cs

```diff
@@ -70,21 +70,13 @@ public static bool IsTooLong(int length)
 
 		public static bool IsTrimable(string? password, out string? trimmedPassword)
 		{
-			trimmedPassword = password;
-			if (password is null)
-			{
-				return false;
-			}
-
-			var beforeTrim = password.Length;
-
-			trimmedPassword = password.Trim();
-
-			if (beforeTrim != trimmedPassword.Length)
+			if (password is {} && password.IsTrimable())
 			{
+				trimmedPassword = password.Trim();
 				return true;
 			}
 
+			trimmedPassword = password;
```

Comment:
Why do we set `trimmedPassword` if `password` isn't trimmable?

Reply:
It must be assigned because it is an `out` value.

Reply:
I would say if `password` is trimmable `trimmedPassword` should be set otherwise `trimmedPassword` should be `null`.

Reply:
Agree with the idea, it would be better from the readability side, but don't want to touch it because there are a few references where the input and the out variable are the same. 

Deamon.cs:
![image](https://user-images.githubusercontent.com/16364053/98525723-29a50580-2279-11eb-92c9-94dde38c4520.png)

It would not be lucky to change the value inside.

Reply:
Although the tests will pass if trimmedPassword is set to null... It would be good the add a test to the case like this.